### ISSUE TITLE
Add armv7 and armv8 host roborio compilers

### ISSUE THREAD

**Issue by ThadHouse** [wpilibsuite/opensdk#34]

**Comment by RyanHir** [comment]
@ThadHouse Are you wanting to compile roboRIO code from a Raspberry Pi or other ARM device? This should be possible. We can bump these builds to either tier 1 or 2. With how common ARM on ARM compilers exist for generic distributions, I would go with tier 2 builds.

**Comment by ThadHouse** [comment]
Yes, for compiling rio code on Pi's. We've had asks before to allow full wpilib development on a pi, and thats one of the next steps necessary there.
